package-res/def/object.js

```diff
@@ -299,10 +299,14 @@ def.copyOwn(def, /** @lends def */{
             to = {}, from  = a, props = b;
 
         if(props) {
-            if(from)
-                props.forEach(function(p) { to[p] = from[p];   });
-            else
-                props.forEach(function(p) { to[p] = undefined; });
+            var i = -1;
+            var L = props.length;
+            if(from) {
+                var prop;
+                while(++i < L) to[(prop = props[i])] = from[prop];
```

Comment:
I'm not sure `prop = props[i]` is evaluated before `from[prop]`. I believe it does (or it wouldn't be working), but it's not clear. I would prefer to do the assignment on a separate statement, or simply use `props[i]` twice.